src/main/java/com/demo/classroom/Entity/Course.java

```diff
@@ -0,0 +1,29 @@
+package com.demo.classroom.Entity;
+
+import java.util.List;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.OneToMany;
+import lombok.AllArgsConstructor;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+
+@Entity
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+public class Course {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long courseId;
+
+    private String title;
+
+    @OneToMany(mappedBy = "course")
+    private List<Book> books;
```

Comment:
Add proper cascade type.
If the relationship is optional (e.g., a Course may or may not have books), ensure the code reflects that.
```suggestion
    @OneToMany(mappedBy = "course")
    private List<Book> books = new ArrayList<>();
```

Reply:
Solved.